VK/VK/Services/Models/NewsPostModels/NewsPostItem.swift

```diff
@@ -0,0 +1,24 @@
+// NewsPostItem.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Пост с новостью
+struct NewsPostItem: Decodable {
+    let id, date: Int
```

Comment:
```suggestion
    let id, date: Int
```
добавь описание всем пропертям